frappe/public/icons/timeless/symbol-defs.svg

```diff
@@ -941,4 +941,10 @@
 		</g>
 	</symbol>
 
+	<symbol viewBox="0 0 13 16" fill="none" xmlns="http://www.w3.org/2000/svg" id="icon-dashboard-list">
```

Comment:
```suggestion
	<symbol viewBox="0 0 13 16" fill="none" xmlns="http://www.w3.org/2000/svg" id="icon-dashboard">
```

Reply:
There is already an icon named 'dashboard'.